### ISSUE TITLE
lua script load return bool true (PHP client)

### ISSUE THREAD

**Issue by jasonhe88** [apache/kvrocks#1132]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.

### Version

MacOS 12.2.1
KVRocks 2.1.0

### Minimal reproduce step

```php
 <?php
  
 $redis = new Redis();
  
 $redis->connect('127.0.0.1', 6379);
  
$r = $redis->script('load', 'return "hello"');
var_dump($r);
```

### What did you expect to see?

hash value of lua script

### What did you see instead?

bool(true)

### Anything Else?

if switch to Redis, above code return correct hash value

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by git-hulk** [comment]
@jasonhe88 Thanks for your feedback. You're right that's a bug which caused by Kvrocks return the inconsistent string with Redis.

The expected result is the bulk string(`$40\r\n{SHA}\r\n`) but returned the simple string(`+{SHA}\r\n`) instead. Refer: https://github.com/apache/incubator-kvrocks/blame/unstable/src/commands/redis_cmd.cc#L5036

**Comment by jasonhe88** [comment]
thank you for your quick response

is there any workaround to resolve this issue?

**Comment by git-hulk** [comment]
@jasonhe88 Are you using `phpredis`? if yes, you can workaround by using the `raw` command and enable the OPT_REPLY_LITERAL option to send the `script load` which will parse the simple string as a string(except in the pipeline or multi mode): https://github.com/phpredis/phpredis/blob/b580505b40ef8b182ca79d49cda38c7d1ea9b5c7/redis.c#L3070


For the raw command, you can have a look at: https://github.com/phpredis/phpredis#rawcommand

For the bug fix can see: #1133

**Comment by jasonhe88** [comment]
Get it, Thank you so much!
